integration/tests/migrations/test_account_migration.py

```diff
@@ -0,0 +1,249 @@
+"""Tests to check old accounts continue work after structure changes
```

Comment:
Could You show where you remove this test from test suites?

Reply:
It is placed on different level
integration/tests/migrations
not basic, not services